src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml

```diff
@@ -66,7 +66,7 @@ services:
     prestashop.core.grid.data_provider.contacts:
         class: '%prestashop.core.grid.data.factory.doctrine_grid_data_factory%'
         arguments:
-            - '@prestashop.core.grid.query_builder.contacts'
+            - '@prestashop.core.grid.query_builder.contact'
             - '@prestashop.core.hook.dispatcher'
             - '@prestashop.core.grid.query.doctrine_query_parser'
             - 'contacts'
```

Comment:
what we could harmonize is the id of the grids too, but it wont be necessary in 1.7.7 as I'll will suggest a new DataCollector will all the information about the displayed grids.

Reply:
Yes, I'll do that too. :+1: